3.12/javascript-api.md

```diff
@@ -104,64 +288,3 @@ The following Node.js modules are not available at all:
 - `tls`
```

Comment:
Let's add links for:
- but see `@arangodb/crypto` below
- but see `@arangodb/request` below